server/moodle/mod/livequiz/mod_form.php

```diff
@@ -55,9 +51,8 @@ public function definition() {
      * @return array of "element_name"=>"error_description" if there are errors,
      *         or an empty array if everything is OK.
      */
-    public function validation($data, $files) {
-        $errors = parent::validation($data, $files);
+    public function validation($data, $files): array {
         // Add custom validation logic here if needed.
-        return $errors;
+        return parent::validation($data, $files);
     }
```

Comment:
```suggestion
    
```
just delete it.
if it all explodes ill fix it

Reply:
Okay my suggestion is fucked up. i dont fully understand how it works.

point is, delete the entire function + the comments and it should be fine